### PR TITLE
Really move to us-central1-b, enable Heapster tests

### PR DESCRIPTION
**Pull request by zmerlynn**
Relies on https://github.com/kubernetes/kubernetes/pull/27395 (and I'll wait for that to finish building)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml [hunk 1]
```diff
@@ -119,30 +119,28 @@
...
                 # TODO: Remove HPA and "master services" tests after enabling Heapster.
-                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Kubernetes\smaster\sservices\sis\sincluded\sin\scluster-info"
+                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
                 export GINKGO_PARALLEL="y"
-                export ZONE="us-east1-a"
+                export ZONE="us-central1-b"
                 export NUM_NODES=2000
                 export MACHINE_TYPE="n1-standard-1"
-                export ALLOWED_NOTREADY_NODES="10"
-                export CLUSTER_IP_RANGE="10.8.0.0/13"
```
Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: I don't understand why we removed CLUSTER_IP_RANGE from here.

@gmarek

Comment (gmarek) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: Default is /14, so removing this should have broken the test.

Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: This is GKE, so default may be different.

Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: OK - I found the corresponding piece in GKE code. So I understand why you removed it :)

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: We handle it automatically now and didn't used to when I first created the job, yeah.

**Pull request merged**